docs/usage/forms.md

```diff
@@ -15,8 +15,31 @@ $post->fill([
 
 To achieve this structure in your form - to prevent manipulating the form data just to save them. You can use the input name array `[]` [syntax](https://www.php.net/manual/en/faq.html.php#faq.html.arrays).
 
-```markup
+```html
 <input type="text" name="en[title]" />
 <input type="text" name="de[title]" />
 ```
 
+# Saving the request directly
```

Comment:
I'm not really happy with this naming. Filling works also with `en[name]`. This naming indicates that this is the preferred approach. For me it isn't - that's personal opinion.^^
At all I would like to keep the docs unopinionated. Could you find a naming that fits better?

Reply:
I'll try to structure the section by `request`, and see what you think about it 👍